core/templates/dev/head/components/review_feedback_thread_modal_directive.html

```diff
@@ -0,0 +1,116 @@
+      </span>
+      <span ng-if="m.updated_subject">
+        <em>Subject changed to '<[m.updated_subject]>'</em>
+      </span>
+    </div>
+
+    <p style="clear: both; text-indent: 4ch"><[m.text]></p>
+
+    <div ng-if="activeThread.isSuggestionThread() && $index == 0">
+      <button ng-click="showSuggestionModal()"
+              class="btn btn-<[getSuggestionButtonType()]> protractor-test-view-suggestion-btn">
+        View Suggestion
+      </button>
+    </div>
+  </li>
+</ol>
+
+<hr style="margin-top: 0">
+
+<div ng-if="userIsLoggedIn" style="padding: 0 32px 32px 32px">
+  <label for="tmpMessageText">
+    Add new message
+    <span ng-if="EditabilityService.isEditable() && !activeThread.isSuggestionThread()">
+      (and/or change status)
+    </span>
+  </label>
+  <textarea ng-model="tmpMessage.text" ng-disabled="messageSendingInProgress"
+            class="form-control protractor-test-feedback-response-textarea"
+            id="tmpMessageText" rows="6">
+  </textarea>
+
+  <div style="margin-top: 12px">
+    <div ng-show="EditabilityService.isEditable() && !activeThread.isSuggestionThread()"
+         style="float: left">
+      Change status (optional):
+      <select ng-model="tmpMessage.status"
+              ng-options="choice.id as choice.text for choice in STATUS_CHOICES"
+              ng-disabled="messageSendingInProgress"
+              style="height: 34px; border-color: #ccc">
+      </select>
+      <div ng-if="!tmpMessage.text && (tmpMessage.status === 'ignored' || tmpMessage.status === 'not_actionable')"
+           style="margin: 8px 0; color: red">
+        <span style="font-size: 19px" class="material-icons">error_outline</span>
+        Please specify a reason for setting the status to "<[getHumanReadableStatus(tmpMessage.status)]>".
+      </div>
+    </div>
+
+    <div class="text-right" style="float: right">
+      <button ng-click="close()" ng-if="activeThread.messages" ng-disabled="messageSendingInProgress"
+              class="btn btn-default protractor-test-oppia-feedback-response-send-btn"
+              style="margin-right: 0">
+        Close
+      </button>
+      <button ng-click="addNewMessage(activeThread.threadId, tmpMessage.text, tmpMessage.status)"
+              ng-disabled="messageSendingInProgress || (!tmpMessage.text && activeThread.status === tmpMessage.status) || (!tmpMessage.text && (tmpMessage.status === 'ignored' || tmpMessage.status === 'not_actionable'))"
+              class="btn btn-success protractor-test-oppia-feedback-response-send-btn"
+              style="margin-left: 8px">
+        <span ng-if="messageSendingInProgress">Sending...</span>
+        <span ng-if="!messageSendingInProgress">Send</span>
+      </button>
+    </div>
+
+    <div style="clear: both"></div>
+
+    <em ng-if="!userIsLoggedIn">To respond to a feedback thread, please log in.</em>
+  </div>
+</div>
+
+
+<style>
+ol.thread-messages {
```

Comment:
Contents of style tags should be indented by 2.

/cc @oppia/dev-workflow-team -- this is a bit odd, I'd expect it to fail lint checks?

Reply:
Done.

Reply:
Hi @seanlip, I checked that there is no lint check present for this.
I will file an issue for this.

Reply:
Thanks! Yep, adding checks sounds good, though I think this is more an HTML thing (2-space indent) than a CSS thing right?